examples/webgl2_multiple_rendertargets.html

```diff
@@ -82,7 +40,7 @@
 				return vec4( mix( pow( value.rgb, vec3( 0.41666 ) ) * 1.055 - vec3( 0.055 ), value.rgb * 12.92, vec3( lessThanEqual( value.rgb, vec3( 0.0031308 ) ) ) ), value.a );
 			}
 
-			layout(location = 0) out vec4 pc_FragColor;
```

Comment:
Btw: does anyone know what "pc" stands for? :thinking:

Reply:
My best guess would be something in the direction of "program compatibility", since `pc_fragColor` is related to GLSL1 compatibility in `WebGLProgram`.

Reply:
Oh yeah, that could be it :+1: 

I just found it weird that I couldn't find anything when I tried looking it up.